source/auth/auth.rst

```diff
@@ -683,6 +683,176 @@ mechanism
 mechanism_properties
 	MUST NOT be specified.
 
+MONGODB-IAM
+~~~~~~~~~~~
+
+:since: 4.4
+
+MONGODB-IAM authenticates using AWS IAM credentials or temporary IAM credentials 
```

Comment:
temporary credentials can also be assigned to a user via AssumeRole

https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html

Reply:
Done

Reply:
Thanks - Please resolve because I cannot